library/ssl_tls13_server.c

```diff
@@ -2885,6 +2890,12 @@ int mbedtls_ssl_tls13_handshake_server_step( mbedtls_ssl_context *ssl )
         /* start state */
         case MBEDTLS_SSL_HELLO_REQUEST:
             mbedtls_ssl_handshake_set_state( ssl, MBEDTLS_SSL_CLIENT_HELLO );
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+            ssl->handshake->tls13_session_tickets =
+                ssl->conf->new_session_tickets ?
```

Comment:
Why do we override the configured number of new session tickets with MBEDTLS_SSL_TLS1_3_DEFAULT_NEW_SESSION_TICKETS here in case it is equal to 0 ?
Otherwise, what about to move that to `ssl_handshake_init` ?

Reply:
It has been moved to `ssl_handshake_init` and remove `override the configured number` .